caffe2/operators/layer_norm_op.cc

```diff
@@ -21,15 +21,16 @@ bool LayerNormOp<CPUContext>::DoRunWithType<float>() {
   auto* mean = Output(1);
   auto* stdev = Output(2);
 
-  CAFFE_ENFORCE_GE(input.dims().size(), 2, "LayerNorm requires input dim >= 2");
+  CAFFE_ENFORCE_GE(
+      input.sizes().size(), 2, "LayerNorm requires input dim >= 2");
```

Comment:
`input.dim()`

Reply:
another refactor recognizing this pattern maybe?

Reply:
maybe after we codemod ndim() to dim()